src/controllers/admin-controller/data-target.controller.ts

```diff
@@ -164,7 +164,7 @@ export class DataTargetController {
     }
 
     @Delete(":id")
-    @ApiOperation({ summary: "Delete an existing IoT-Device" })
+    @ApiOperation({ summary: "Delete an existing data-target" })
```

Comment:
Be consistent with the naming. On the other controller endpoints, "DataTarget" is used.